py/server/deephaven/csv.py

```diff
@@ -45,7 +45,7 @@ def read(
         skip_rows (long): number of data rows to skip before processing data. This is useful when you want to parse
             data in chunks. Defaults to 0
         num_rows (long): max number of rows to process. This is useful when you want to parse data in chunks.
-            Defaults to {@link Long#MAX_VALUE}
+            Defaults to `Long#MAX_VALUE <https://docs.oracle.com/javase/8/docs/api/java/lang/Long.html#MAX_VALUE>`_
```

Comment:
```suggestion
            Defaults to the maximum 64bit integer value
```
Trying to minimize java references.